openmrs/apps/reports/RADET/sql/index_testing.sql

```diff
@@ -1,33 +1,35 @@
-SELECT 0 as "Contacts  (Contact nr/Child nr) ",
-	0 as "Date of Contact elicitation from Index/ Date de sollicitation du contact à partir de l'index",
-	0 as "Date of birth (Date de naissance)",
+SELECT getPatientIdentifier(p.patient_id) as "Contacts (Contact nr/Child nr) ",
+	getPatientIndexTestingDateOffered(getFirstIndexID(p.patient_id)) as "Date of Contact elicitation from Index/ Date de sollicitation du contact à partir de l'index",
+	getPatientBirthdate(p.patient_id) as "Date of birth (Date de naissance)",
 	"" as "Current Age (Âge actuel)",
 	"" as "Age (in months  for less than 5yrs) / Âge (en mois pour moins de 5 ans) ",
 	"" as "Age Groups (Les groupes d'âge )",
-	0 as "Sex",
-	0 as "Index Case Code (ART Code/ Testing Code*)",
-	0 as "Type of Index Case/ type de cas index",
-	0 as "Index Classification / Classification de l'indice",
-	0 as "HTS Entry/ entrée  (Testing Modality/ modalités de test)",
-	0 as "Date of contact notification / Date de notification de contact",
-	0 as "Method of contact notification/ Méthode de notification de contact",
-	0 as "Outcomes of notifictaion / Résultats de la notification",
-	0 as "HTS Client ID (EID or HEI Number)",
-	0 as "Actual date of visit for HTS/ Date réelle de visite pour HTS (EID Sample Collected date/Échantillon EID Date de collecte)  (dd-MMM-yyyy)",
-	0 as "HTS Results/ Résultats HTS ",
-	0 as "Location of testing/ emplacement des tests",
-	0 as "Date Enrolled to HIV care / Date d'inscription aux soins du VIH (dd-MMM-yyyy)",
-	0 as "Linked to ART / Lié à ART",
-	"" as "Client ART Unique ID/Patient Unique ID/ART TARV",
-	0 as "ART Start Date/ Date de début de l'ART (dd-MMM-yyyy)",
-	0 as "Name of Facility where Treatment is provided/ Nom de l'établissement où le traitement est dispensé",
-	0 as "Reasons for Non-Linkage/ Raisons du non-couplage",
-	"" as "Time to Enrolment (days)",
-	"" as "Time to ART Initiation   (days)",
-	0 as "Date offered ICT / Date offerte ICT",
-	0 as "Date accepted ICT / Date d'acceptation ",
-	0 as "No of contacts elicited/ Nombre de contacts obtenus",
-	0 as "No of contacts notified / Nombre de contacts notifiés",
-	0 as "No of contacts tested for HIV/ Nombre de contacts testés pour le VIH",
-	0 as "No of contacts tested HIV positive/ Nombre de contacts testés séropositifs",
-	0 as "APS";
\ No newline at end of file
+	getPatientGender(p.patient_id) as "Sex",
+	getPatientARTNumber(p.patient_id) as "Index Case Code (ART Code/ Testing Code*)",
```

Comment:
@lauravignoli is this meant to be the ART number of the index related to the current contact? (i.e. `getPatientARTNumber(getFirstIndexID(p.patient_id))` )

Reply:
@tmvumbi2 I think the index ARV number goes in line 21 getPatientARTNumber(getFirstIndexID(p.patient_id)) as "Client ART Unique ID/Patient Unique ID/ART TARV",

Reply:
@lauravignoli  ah okay I see, thanks :+1: